src/sql/workbench/browser/designer/designer.ts

```diff
@@ -941,6 +983,56 @@ export class Designer extends Disposable implements IThemable {
 		return component;
 	}
 
+	private addTableDesignerTaskbar(container: HTMLElement, tableProperties: DesignerTableProperties,
```

Comment:
this is the designer component, we shouldn't have the word `TableDesigner` in it.